siteConfig.yaml

```diff
@@ -0,0 +1,64 @@
+site:
+  navBar:
+    templateId: "standard"
+    navigationLinks:
+      - title: "Courses"
+        link: "/#courses"
+      - title: "Events"
+        link: "/events"
+      - title: "Blog"
+        link: "/blog"
+
+  brand:
+    name: "TORQBIT"
+    logo: "/icon/torqbit.png"
+    title: "A platform to guide you"
+    description: "Master the art of product building and turn your ideas into successful, scalable products"
+    ogImage: "/icon/torqbit.png"
+    favicon: "/favicon.ico"
+    brandColor: "#6B74EE"
+
+  darkMode: true
+
+  heroSection:
+    templateId: "standard"
+    title: "The Awesomest teaching platform"
+    description: >
+      Torqbit, an open-source platform, serves as a learning management software
+      that works for single person to largest educational institutions
+    actionButtons:
+      primary:
+        label: "Get Started"
+        link: "/dashboard"
+      secondary:
+        label: "Contact Us"
+        link: "mailto:support@torqbit.com"
+    banner:
+      lightModePath: "/img/landing/online-learning.png"
+      darkModePath: "/img/landing/online-learning.png"
+      position: "right"
+
+  footer:
+    discordLink: "https://discord.gg/NserMXcx"
+    githubLink: "https://github.com/torqbit"
+    youtubeLink: "https://www.youtube.com/@torqbit"
```

Comment:
better to replace `discordLink` with just `discord` and similarly others. Also i am thinking of moving the social links to brand section, as it is part of it.